### PR TITLE
Update What's New for 2.4.1 GA docs

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) adds 2.4.1 doc updates to the What's New list.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable: 2.4.1

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/whats-new.html

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,99 @@ description:
...
+  link: https://github.com/magento/merchdocs/pull/758
+- description: Enhanced [Security Scan](https://docs.magento.com/user-guide/magento/security-scan.html) runs over 17,000 tests to monitor your sites
+    for potential risks, with real-time and historical reports and notifications.
+  versions: 2.4.1
+  type: Minor update
+  date: October 15, 2020
+  link: https://github.com/magento/merchdocs/pull/748
+- description: "[Customer Configuration - CAPTCHA](https://docs.magento.com/user-guide/configuration/customers/customer-configuration.html)
+    has a new option under Forms that displays CAPTCHA in the storefront during checkout
+    for registered customers."
```
Comment (erikmarr) [inline src/_data/whats-new.yml:57]:
```suggestion
    for registered customers.
```

File: src/_data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,99 @@ description:
...
+  versions: 2.4.1
+  type: Minor update
+  date: October 15, 2020
+  link: https://github.com/magento/merchdocs/pull/745
+- description: "[Yotpo Reviews](https://docs.magento.com/user-guide/marketing/yotpo-reviews-intro.html#changes-in-the-latest-release)
+    module enhancement to programmatically  adjust when Magento orders are synced
```
Comment (erikmarr) [inline src/_data/whats-new.yml:63]:
```suggestion
    module enhancement to programmatically adjust when Magento orders are synced
```

File: src/_data/whats-new.yml [hunk 3]
```diff
@@ -4,8 +4,99 @@ description:
...
+  versions: 2.4.1
+  type: New topic
+  date: October 15, 2020
+  link: https://github.com/magento/merchdocs/pull/681
+- description: Added the "Add from Cart" method to [My Requisition Lists](https://docs.magento.com/user-guide/customers/account-dashboard-requisition-lists.html).
+  versions: 2.4.1
+  type: Major update
+  date: October 15, 2020
+  link: https://github.com/magento/merchdocs/pull/655
+- description: "[Media Gallery Asset Management](https://docs.magento.com/user-guide/cms/media-gallery-asset-management.html)
```
Comment (erikmarr) [inline src/_data/whats-new.yml:81]:
```suggestion
- description: [Media Gallery Asset Management](https://docs.magento.com/user-guide/cms/media-gallery-asset-management.html)
```

File: src/_data/whats-new.yml [hunk 4]
```diff
@@ -4,8 +4,99 @@ description:
...
+    was updated to include expanded asset details, as well as new asset search and
+    image metadata editing."
```
Comment (erikmarr) [inline src/_data/whats-new.yml:83]:
```suggestion
    image metadata editing.
```

File: src/_data/whats-new.yml [hunk 5]
```diff
@@ -4,8 +4,99 @@ description:
   We exclude from this list proofreading, spelling checks, and insignificant updates.
 link: /getting-started/whats-new.html
 thread: /whatsnew-feed.xml
-updated: Wed Aug 26 12:49:06 2020
+updated: Tue Oct 13 12:32:01 2020
 entries:
+- description: Updates the Page Builder [Banner](https://docs.magento.com/user-guide/cms/page-builder-media-banner.html#complex-content-in-the-settings)
+    and [Slider](https://docs.magento.com/user-guide/cms/page-builder-media-slider.html#complex-content-in-the-settings) topics for the change to no longer
```
Comment (erikmarr) [inline src/_data/whats-new.yml:10]:
```suggestion
    and [Slider](https://docs.magento.com/user-guide/cms/page-builder-media-slider.html#complex-content-in-the-settings) topics to no longer
```

File: src/_data/whats-new.yml [hunk 6]
```diff
@@ -4,8 +4,99 @@ description:
...
+    allow links to be added to both the Message Text area and the Link attribute for Banners and Slides.
```
Comment (erikmarr) [inline src/_data/whats-new.yml:11]:
```suggestion
    allow links in the Message Text and Link attribute fields for Banners and Slides.
```

**Comment by leslietilling** [comment]
running tests

**Pull request merged**